map_test.go

```diff
@@ -2213,12 +2221,15 @@ func ExampleMap_perCPU() {
 		for _, n := range values {
 			sum += n
 		}
-		fmt.Printf("Sum of %d: %d\n", key, sum)
+		fmt.Printf("Mean of %d: %d\n", key, sum/uint32(possibleCpus))
```

Comment:
Not sure if calculating the mean is more illustrative of what per-cpu maps are generally used for. Sure, this test needs to run on machines of any core count, but then we should probably stick to updating only the first two CPUs like the example originally did.

Per-cpu maps are typically used for keeping packet/event counters without requiring atomics on the bpf side, and their contents are usually summed to gather total results. I don't think this new proposal is strictly clearer for a newcomer. Per-cpu maps are also seldom written to, so introducing `MustPossibleCPU()` in this context may be a little distracting.

Reply:
> Per-cpu maps are typically used for keeping packet/event counters

hah, yes that's exactly what I'm using them for.

Instead of the mean trick, I was thinking I could emit the *last* value, which could be the same as the first value for cpus==1

Reply:
But otherwise it's very hard to generate testable output here.

Reply:
Okay, to illustrate the width of the value then, let's use PossibleCPUs() to set all cpus' slots to the same value, and in the nested loop ( currently `for _, n := range values {`), we can assert that all slots contain the same value instead of summing them. A bit contrived, but relatively straightforward.

I'm on the fence about limiting the example to only a single key, but you can make that call. We could add a comment about only using a single key for demonstration purposes in case there's any doubt.

Reply:
It does mean we can't use the sum in the output.